actions/contacts/routes.go

```diff
@@ -18,12 +18,11 @@ func NewHandler(appConfig *config.AppConfig, services *config.AppServices) route
 
 	apiEndpoints := routes.APIEndpointsFunc(func(router *gin.RouterGroup) {
 		contactGroup := router.Group("/contact")
-		contactGroup.POST("", action.create)
-		contactGroup.PATCH("", action.update)
+		contactGroup.PUT("/{paymail}", action.upsert)
```

Comment:
It should be probably something like:
contactGroup.PUT("/:paymail", action.upsert

but it still doesn't respond to:
curl -X 'PUT' \
  'http://localhost:3003/v1/contact/alice%40wojtek.4chain.space' \
  -H 'accept: application/json' \
  -H 'x-auth-xpub: xpub...' \
  -H 'Content-Type: application/json' \
  -d '{
  "fullName": "xyz xyz"
}'

Reply:
https://gin-gonic.com/docs/examples/param-in-path/

Reply:
fixed